internal/errors/code.go

```diff
@@ -22,12 +22,15 @@ const (
 
 	// General function errors are reserved Codes 100-999
 	InvalidParameter Code = 100 // InvalidParameter represents and invalid parameter for an operation.
+	GenerateId       Code = 101
```

Comment:
New codes to avoid a generic `internal` or similar error and to avoid using `unknown`